### PR TITLE
Factor out the style.css

### PR DESCRIPTION
**Pull request by Lonami**

### CODE REVIEW

**Comment by Lonami** [comment]
Not sure if the `<meta http-equiv="refresh" content="0;../index.html" />` in `en/index.html` is the best way to handle this, ideally it would be a redirect in the web server, but it's the only way I can think of without duplicating the file because the `.css` path would now be different (unless we made another symbolic link to that?).

File: ar/index.html [hunk 1]
```diff
@@ -6,76 +6,9 @@
...
+  <link rel="stylesheet" type="text/css" href="../style.css">
   <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
   <meta name="viewport" content="width=device-width، initial-scale=1، shrink-to-fit=no">
-    <style>
-    html {
-      height: 100%;
-    }
-    *{
-      text-align: right!important;
-    }
```
Comment (tulir) [inline ar/index.html:17]: I think this was a custom style specifically for arabic

File: ko/index.html [hunk 2]
```diff
@@ -6,82 +6,9 @@
...
-      line-height: 1.6;
-      font-size: 18px;
-      color: #222;
-      background-color: #FAFAFA;
-      padding: 0 10px;
-      font-family: "Noto Sans KR";
-    }
-    em {
-      font-style: normal;
-    }
```
Comment (tulir) [inline ko/index.html:32]: This is also a custom style

**Comment by Lonami** [comment]
> The CSS link could just be absolute instead of relative

Then it will break offline, which I dislike, but it's fair enough.

**Comment by Lonami** [comment]
There were actually a few more differences. I don't know if `!important` is needed, but it works.

**Pull request merged**